src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -93,13 +93,16 @@ def get(self, request: Request, project: Project, event_id: str) -> Response:
             # Exception is already source mapped
             return self._create_response()
 
+        frame, filename, abs_path = self._get_frame_filename_and_path(exception, frame_idx)
```

Comment:
nit: usually I would prefer to try parsing the `frame_idx` into an `int` instead of passing the string and coercing it in the function.